### PR TITLE
tar: explain [a] flag more clearly

### PR DESCRIPTION
**Pull request by Seppel3210**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/tar.md [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 `tar xf {{path/to/source.tar[.gz|.bz2|.xz]}} --directory={{path/to/directory}}`
 
-- [c]reate a compressed archive and write it to a [f]ile, using [a]rchive suffix to determine the compression program:
+- [c]reate a compressed archive and write it to a [f]ile, using file suffix to [a]utomatically determine the compression program:
```
Comment (gutjuri) [inline pages/common/tar.md:27]:
```suggestion
- [c]reate a compressed archive and write it to a [f]ile, using filename's suffix to [a]utomatically determine the compression program:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks, I have added one suggestion to make it clear that in fact the file**name**'s suffix is used for detection.

File: pages/common/tar.md (hunk 1 above)
Comment (Seppel3210) [reply pages/common/tar.md]: I kinda forgot this PR 😅 but I think "file extension" is better and more commonly used

Comment (gutjuri) [reply pages/common/tar.md]: You're totally right!

**Review by gutjuri — APPROVED** [review]
Thank you!

**Pull request merged**